lib/screens/authenticate/sign_in.dart

```diff
@@ -22,81 +24,68 @@ class _SignInState extends State<SignIn> {
   String email = '';
   String password = '';
 
+  Future<void> handleLogin() async {
+    if (_formKey.currentState.validate()) {
+      setState(() => loading = true);
```

Comment:
After pressing login, it shows the yellow/brown loading screen. This should not be there.

Reply:
At this branch, the loading screen is not redesigned yet
At upd-design-2 it is

Reply:
Fine.